aiosmtpd/tests/test_smtp.py

```diff
@@ -80,7 +101,14 @@ async def auth_DONT(
     ):
         return MISSING
 
-    async def auth_WITH_UNDERSCORE(self, server, args):
+    async def auth_WITH_UNDERSCORE(self, server: Server, args):
+        """
+        Be careful when using this AUTH mechanism; log_client_response is set to
+        True, and this will raise some severe warnings.
+        """
+        await server.challenge_auth(
+            "challenge", encode_to_b64=False, log_client_response=True
+        )
```

Comment:
The only reason is to test / cover the `log_client_response=True` path of `SMTP.challenge_auth()`

I was a bit lazy to write yet-another-auth-mechanism (because I'll then have to edit several places), so I just 'repurpose' existing one.